### PR TITLE
Accessibility improvements

### PR DESCRIPTION
**Pull request by mojulia**
Accessibility improvements as suggested by the #shelter accessibility audit.

### CODE REVIEW

File: templates/pgbar.tpl.php [hunk 1]
```diff
@@ -26,7 +26,7 @@
 ?>
 <div id="<?php print $html_id; ?>" class="pgbar-wrapper progress-wrapper" data-pgbar-current="<?php print $current; ?>" data-pgbar-target="<?php print $target; ?>">
   <p><?php print $intro_message; ?></p>
-  <div class="progress" role="progressbar" tabindex="0" aria-valuenow="<?php echo $percentage; ?>" aria-valuemin="0" aria-valuemax="100">
+  <div class="progress" role="progressbar" aria-valuenow="<?php echo $percentage; ?>" aria-valuemin="0" aria-valuemax="100">
```
Comment (katzmo) [inline templates/pgbar.tpl.php]: I think we should probably also replace the "progressbar" role with "meter". :thinking: 

* According to MDN a progressbar "indicates that a request has been received and the application is making progress toward completing the requested action." – not what this kind of progress bar does.
* Meter sounds more general: "A meter is a graphical display of a numeric value within a defined range." ([MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/meter_role))

Both require a label which we currently do not provide…

Comment (mojulia) [reply templates/pgbar.tpl.php]: Sounds like it should be replaced yes! You mean we'd need to define an `aria-label` that in a way describes the progress bar more clearly? Or something else?

Comment (katzmo) [reply templates/pgbar.tpl.php]: I mean that the `aria-label` issue stays the same, no matter if we use "progressbar" or "meter" :wink:

I don’t see an easy way to implement it, we can’t just hard-code some label. It needs to accurately describe the progressbar/meter and be localized.

Comment (mojulia) [reply templates/pgbar.tpl.php]: Ah so we might as well change the role to "meter" now as there already is no label? ;) 

I'm afraid I'm not clear what the next steps would be - we do use `aria-label` in other places, how did we come up with those and are they localised? :thinking: [This bascially is point 4 here, right?](https://www.notion.so/more-onion/eaf6b06080184e54ab282c2d3446c343?v=b446986fc62b4fe5a41d3b5d876ed65a&p=5daa558b294e40d7b46a80120baa7031&pm=s)

Comment (katzmo) [reply templates/pgbar.tpl.php]: Think so, although point 4 says "name". 

`aria-label` points to another element on the page that acts as a label. Only in the case of the pgbar, there is no element that provides any label. 

I think it’s ok for this PR to switch to "meter" but not add a label for now, as this still needs more discussion.

Comment (mojulia) [reply templates/pgbar.tpl.php]: sounds good, point 4 is also on the "still needs discussion"-list :) but will change the role! thanks!

**Review by katzmo — APPROVED** [review]

**Pull request merged**